Makefile

```diff
@@ -497,13 +499,13 @@ $(JS_DEST): $(JS_SOURCES) | node_modules
 	@touch $(JS_DEST)
 
 .PHONY: fomantic
-fomantic: node-check $(FOMANTIC_DEST_DIR)
+fomantic: node-check $(FOMANTIC_EVIDENCE)
 
-$(FOMANTIC_DEST_DIR): semantic.json web_src/fomantic/theme.config.less | node_modules
+$(FOMANTIC_EVIDENCE): semantic.json web_src/fomantic/theme.config.less web_src/fomantic/_site/globals/* | node_modules
```

Comment:
Can `make` actually resolve wildcards in the prereqs? I thought it could not because we use `find` to resolve them beforehand for other targets.

Reply:
@silverwind You're right, it was warking in my system, but it seems it's not universal. The proper way to solve that is to use `$(wildcard pattern...)`. Fixed.

Reply:
Just found out: rather than not being universal, the reason simple wildcards are not recommended is [this](https://www.gnu.org/software/make/manual/html_node/Wildcard-Pitfall.html#Wildcard-Pitfall). A `*` will not expand if there are no files (and remain `*`, which is not an existing file), but the `$(wildcard...)` function will expand to empty.